docs/glossary.md

```diff
@@ -0,0 +1,23 @@
+---
+id: Glossary
+title: Backstage Glossary
+# prettier-ignore
+description: List of all abbreviations, terms, phrases used in Backstage together with its explanations.
+---
+
+# Backstage Glossary
```

Comment:
In practice, this will create a duplicate header, with line 3 displaying at the top and this right under it. Suggest this should be either Overview, or my suggestion, removed entirely?

```suggestion
```